app/main.py

```diff
@@ -1,5 +1,72 @@
+from typing import List
+
+
 class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(
+            self,
+            name: str,
+            weight: int,
+            coords: List[int] = None
+    ) -> None:
+        self.name = name
+        self.weight = weight
+        self.coords = coords if coords is not None else [0, 0]
```

Comment:
You could simplify this condition
```suggestion
        self.coords = coords if coords else [0, 0]
```